src/physics/em/detail/RayleighInteractor.i.hh

```diff
@@ -31,8 +31,6 @@ RayleighInteractor::RayleighInteractor(const RayleighNativeRef& shared,
     , inc_direction_(direction)
     , element_id_(el_id)
 {
-    CELER_EXPECT(inc_energy_ >= this->min_incident_energy()
```

Comment:
Is this removal intentional?  It does not seem to be noted in the git log.

Reply:
Ah I removed that because inferred the min/max energy accessors from Rayleigh, which originally were in the other interactors as well